sync/shared/variables.yaml

```diff
@@ -1,5 +1,5 @@
 # Kubernetes version, pick major and minor, patch will be the latest released.
-kubernetes_version: "1.24"
+kubernetes_version: "1.23"
```

Comment:
This is the more stable latest version we have working
Since 1.24 remove the dockershim and `--network-plugin` flags, breaking the nssm and `kubeadm join`.